crates/matrix-sdk-ui/src/timeline/day_dividers.rs

```diff
@@ -855,4 +864,61 @@ mod tests {
         assert!(iter.next().unwrap().is_remote_event());
         assert!(iter.next().is_none());
     }
+
+    #[test]
+    fn test_dayly_divider_mode() {
+        let mut items = ObservableVector::new();
+        let mut txn = items.transaction();
+
+        let mut meta = test_metadata();
+
+        txn.push_back(meta.new_timeline_item(event_with_ts(MilliSecondsSinceUnixEpoch(uint!(0)))));
+        txn.push_back(
+            meta.new_timeline_item(event_with_ts(MilliSecondsSinceUnixEpoch(uint!(100000000)))),
+        );
+        txn.push_back(meta.new_timeline_item(event_with_ts(MilliSecondsSinceUnixEpoch::now())));
```

Comment:
I am uncomfortable with the use of `now` in a test. I was trying to think of a reason why, but the best I can come up with is what if we want to run the tests on a non-Internet-connected device that does not have a battery-backed clock, so it thinks it's 1970.

Seems unlikely, but is there a good reason why you _want_ to use `now`?

Reply:
Haha, but no, there's no particular reason apart from writing conveninence. I'll change it into something more reasonable 👍